pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBuilderImpl.java

```diff
@@ -122,6 +122,13 @@ private CompletableFuture<Boolean> checkDlqAlreadyExists(String topic) {
                     || actEx instanceof PulsarClientException.TopicDoesNotExistException
                     || actEx instanceof PulsarAdminException.NotFoundException) {
                 existsFuture.complete(false);
+            } else if (actEx instanceof PulsarClientException.NotSupportedException) {
```

Comment:
Same as the above comment, can we just check if the new API is supported by the broker and decide to use the new API or old API?

Reply:
Done